Android/PowerUpApp/hs_err_pid18659.log

```diff
@@ -0,0 +1,521 @@
+#
```

Comment:
Do not commit your logs to the repo. Amend the commit removing this file using `git commit --amend`